runtime/core/decoder/torch_asr_decoder.cc

```diff
@@ -272,6 +277,28 @@ void TorchAsrDecoder::UpdateResult() {
   }
 }
 
+float TorchAsrDecoder::LeftToRightScore(const torch::Tensor& prob,
+                                        const std::vector<int>& hyp,
+                                        const int& eos) {
```

Comment:
for basic type, just use value passing

Reply:
use accessor instead of .item<float>() to access item, which is more efficient.
please refer
https://github.com/wenet-e2e/wenet/blob/main/runtime/core/decoder/ctc_wfst_beam_search.cc#L68